src/engraving/libmscore/jump.cpp

```diff
@@ -54,12 +54,12 @@ const std::vector<JumpTypeTableItem> jumpTypeTable {
     { JumpType::DS_AL_FINE, "D.S. al Fine", "segno", "fine", "" },
     { JumpType::DS,         "D.S.",         "segno", "end",  "" },
 
-    { JumpType::DC_AL_DBLCODA,  "D.C. al Double Coda",   "start", "varcoda",  "codab" },
-    { JumpType::DS_AL_DBLCODA,  "D.S. al Double Coda",   "segno", "varcoda",  "codab" },
-    { JumpType::DSS,            "Dal Segno Segno",       "varsegno", "end",  "" },
-    { JumpType::DSS_AL_CODA,    "D.S.S. al Coda",        "varsegno", "coda",  "codab" },
-    { JumpType::DSS_AL_DBLCODA, "D.S.S. al Double Coda", "varsegno", "varcoda", "codab" },
-    { JumpType::DSS_AL_FINE,    "D.S.S. al Fine",        "varsegno", "fine",  "" },
+    { JumpType::DC_AL_DBLCODA,  "D.C. al Codetta",   "start", "varcoda",  "codab" },
+    { JumpType::DS_AL_DBLCODA,  "D.S. al Codetta",   "segno", "varcoda",  "codab" },
+    { JumpType::DSS,            "Dal Segno Segno",   "varsegno", "end",  "" },
+    { JumpType::DSS_AL_CODA,    "D.S.S. al Coda",    "varsegno", "coda",  "codab" },
+    { JumpType::DSS_AL_DBLCODA, "D.S.S. al Codetta", "varsegno", "varcoda", "codab" },
+    { JumpType::DSS_AL_FINE,    "D.S.S. al Fine",    "varsegno", "fine",  "" },
```

Comment:
@oktophonie, what do you think of these?

Reply:
I'm not familiar enough with the arcane mysteries of either double codas nor with Italian to say. The language mixture is certainly a bit odd, but whether these changes are 'correct' I don't know.

Reply:
I'm not sure either, only thing I am sure about is that the Double Coda was called Codetta in Mu1.
And that there is neither Codetta nor Double Coda as a separate/single qlyph in SMuFL

Reply:
We could also change "Double" to "Doppio", to avoid that language mix.
I don't really care either way.

Reply:
@mike-spa, what do you think about 'Double Coda', 'Doppio Coda' and 'Codetta'?

Reply:
Ah, yes, we do have an Italian in the team ;-)